lib/tdf3/index.ts

```diff
@@ -14,6 +14,7 @@ import {
   type EncryptResult,
   type PemKeyPair,
 } from './src/crypto/declarations.js';
+export * as WebCryptoService from './src/crypto/index.js';
```

Comment:
Nit, but I'd prefer all the exports being on the bottom of the file. 

I can make the change.

Reply:
done